cmd/kubeadm/app/master/apiclient.go

```diff
@@ -271,9 +269,11 @@ func createDummyDeployment(client *clientset.Clientset) {
 		return true, nil
 	})
 
-	fmt.Println("<master/apiclient> test deployment succeeded")
+	fmt.Println("[apiclient] Test deployment succeeded")
 
-	if err := client.Extensions().Deployments(api.NamespaceSystem).Delete("dummy", &v1.DeleteOptions{}); err != nil {
```

Comment:
Have you checked if https://github.com/kubernetes/kubernetes/pull/38330 fixes your problem? Can we just put a priority on getting that merged?